opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py

```diff
@@ -511,6 +511,9 @@ def start_span(  # pylint: disable=too-many-locals
         )
 
         if sampling_decision.sampled:
+            context.trace_options = trace_api.TraceOptions.get_sampled(
```

Comment:
`get_sampled` is used with an argument only once, here. Instead of having that class function, why don't just do the OR operation here?

Reply:
Performing an OR operation here would imply exposing the fact that `TraceOptions` is a bitmask.  I want to keep this an internal implementation detail of `TraceOptions`.

Reply:
Ok, I changed it to use the OR operation directly here, I still don't like it but I cannot find a better solution.

Reply:
Ok, I understand your concern about hiding this as an implementation detail. I think this can be enhanced later and will require a bit of a refactor of the `TraceOptions` API. In the meantime, I am approving this changes to make this PR move forward specially considering that this is a bug fix.